build.gradle

```diff
@@ -14,13 +14,11 @@ repositories {
 }
 
 dependencies {
```

Comment:
Sometimes I encountered while removing certain dependencies:

```General error during semantic analysis: Unsupported class file major version 63```

This is due to incompatible Gradle and JVM versions. Gradle 6.4 is compatible with JVM 14- (https://docs.gradle.org/current/userguide/compatibility.html)